llvm/lib/Target/AMDGPU/AMDGPULegalizerInfo.cpp

```diff
@@ -350,6 +350,12 @@ static std::initializer_list<LLT> AllS32Vectors = {
 static std::initializer_list<LLT> AllS64Vectors = {V2S64, V3S64, V4S64, V5S64,
                                                    V6S64, V7S64, V8S64, V16S64};
 
+static std::initializer_list<LLT> AllVectors{
```

Comment:
should all of these lists be static const?

Reply:
Maybe even `constexpr`?

Reply:
https://github.com/llvm/llvm-project/pull/131673